src/utils/index.ts

```diff
@@ -160,45 +82,48 @@ function toRippledAmount(amount: RippledAmount): RippledAmount {
   };
 }
 
-function convertKeysFromSnakeCaseToCamelCase(obj: any): any {
-  if (typeof obj === "object") {
-    const accumulator = Array.isArray(obj) ? [] : {};
-    let newKey;
-    return Object.entries(obj).reduce((result, [key, value]) => {
-      newKey = key;
-      // taking this out of function leads to error in PhantomJS
-      const FINDSNAKE = /([a-zA-Z]_[a-zA-Z])/g;
-      if (FINDSNAKE.test(key)) {
-        newKey = key.replace(FINDSNAKE, (r) => r[0] + r[2].toUpperCase());
-      }
-      result[newKey] = convertKeysFromSnakeCaseToCamelCase(value);
-      return result;
-    }, accumulator);
-  }
-  return obj;
-}
+/**
+ * Removes undefined values from an object.
+ *
+ * @param obj - Object to remove undefined values from.
+ * @returns The same object, but without undefined values.
+ */
+function removeUndefined<T extends Record<string, unknown>>(obj: T): T {
+  const newObj = { ...obj };
+
+  Object.entries(obj).forEach(([key, value]) => {
+    if (value == null) {
+      /* eslint-disable-next-line @typescript-eslint/no-dynamic-delete -- Deletes undefined values. */
+      delete newObj[key];
+    }
+  });
 
-function removeUndefined<T extends object>(obj: T): T {
-  return _.omitBy(obj, (value) => value == null) as T;
+  return newObj;
 }
 
 /**
```

Comment:
All these time methods could be moved to a separate file called `timeConversion` or something